client/lib/url/support.js

```diff
@@ -12,7 +12,7 @@ export default {
 	CHANGE_NAME_SERVERS_FINDING_OUT_NEW_NS: `${root}/domains/change-name-servers/#finding-out-your-new-name-server`,
 	COMMENTS: `${root}/category/comments`,
 	COMMUNITY_TRANSLATOR: `${root}/community-translator`,
-	CONTACT: `https://wordpress.com/help/contact`,
+	CONTACT: `/help/contact`,
```

Comment:
Probably a solid idea. I can rename it and then update the places we're currently using it.